R/get_message.R

```diff
@@ -0,0 +1,40 @@
+# get_message ------------------------------------------------------------------
```

Comment:
could you add documentation (and `#' @noRd`) of the parameters, in particular the ellipsis? It's clear to me now, but it'd be good to have it written down.